<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="battery">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="contact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="treatment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Treatment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contact_lost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact Lost</string>
    </property>
   </widget>
   <widget class="QPushButton" name="power">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="display">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QProgressBar" name="session_progress">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="new_session">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white</string>
     </property>
     <property name="text">
      <string>NEW SESSION</string>
     </property>
    </widget>
    <widget class="QLabel" name="session_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white</string>
     </property>
     <property name="text">
      <string>SESSION LOG</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white</string>
     </property>
     <property name="text">
      <string>TIME AND DATE</string>
     </property>
    </widget>
    <widget class="QLabel" name="timer">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white</string>
     </property>
     <property name="text">
      <string>0:00</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up_arrow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_arrow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>❚❚</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>◻</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
